evaluator/autoware_control_evaluator/include/autoware/control_evaluator/metrics/deviation_metrics.hpp

```diff
@@ -42,6 +42,30 @@ double calcLateralDeviation(const Trajectory & traj, const Point & point);
  */
 double calcYawDeviation(const Trajectory & traj, const Pose & pose);
 
+/**
+ * @brief calculate longitudinal deviation of the given pose and point
+ * @param [in] pose input pose
+ * @param [in] point input point
+ * @return longitudinal deviation
+ */
+double calcLongitudinalDeviation(const Pose & base_pose, const Point & target_point);
+
+/**
+ * @brief calculate lateral deviation of the given pose and point
+ * @param [in] pose input pose
+ * @param [in] point input point
+ * @return lateral deviation
+ */
+double calcLateralDeviation(const Pose & base_pose, const Point & target_point);
+
+/**
+ * @brief calculate yaw deviation of the given poses
+ * @param [in] pose input pose
+ * @param [in] pose input pose
+ * @return yaw deviation
+ */
+double calcYawDeviation(const Pose & base_pose, const Pose & target_pose);
```

Comment:
Please write the details that you are calculating the distance from the base pose to the target point. p

Reply:
```suggest
/**
 * @brief calculate longitudinal deviation from target_point to base_pose
 * @param [in] pose input base_pose
 * @param [in] point input target_point
 * @return longitudinal deviation from base_pose to target_point
 */
double calcLongitudinalDeviation(const Pose & base_pose, const Point & target_point);

/**
 * @brief calculate lateral deviation from target_point to base_pose
 * @param [in] pose input base_pose
 * @param [in] point input target_point
 * @return lateral deviation from base_pose to target_point
 */
double calcLateralDeviation(const Pose & base_pose, const Point & target_point);

/**
 * @brief calculate yaw deviation from base_pose to target_pose
 * @param [in] pose input base_pose
 * @param [in] pose input target_pose
 * @return yaw deviation from base_pose to target_pose
 */
double calcYawDeviation(const Pose & base_pose, const Pose & target_pose);
```
Thank you very much !
I updated as above !